src/panels/developer-tools/statistics/developer-tools-statistics.ts

```diff
@@ -112,16 +112,10 @@ class HaPanelDevStatistics extends SubscribeMixin(LitElement) {
         filterable: true,
         direction: "asc",
         width: "30%",
-        template: (issues) =>
-          html`${issues
-            ? issues.map(
-                (issue) =>
-                  localize(
-                    `ui.panel.developer-tools.tabs.statistics.issues.${issue.type}`,
-                    issue.data
-                  ) || issue.type
-              )
-            : localize("ui.panel.developer-tools.tabs.statistics.no_issue")}`,
+        valueColumn: "issues_string",
+        template: (_, data: any) =>
```

Comment:
Change the key name of this column to `issues_string` instead and use:

`template: (issues_string) =>`

Reply:
updated